project-base/storefront/helpers/errors/applicationErrors.ts

```diff
@@ -1,28 +1,50 @@
+export type ApplicationErrorVerbosityLevel = 'flash-message' | 'no-flash-message' | 'no-log';
+
 export const ApplicationErrors = {
```

Comment:
Since we have those nice typings everywhere, would be great to have types also for type of error.

```suggestion
export const ApplicationErrors: Record<string, ApplicationErrorVerbosityLevel > = {...}
```

Reply:
I had this typed as you suggested, but after doing so, all three subtypes (`FlashMessageKeys`, `NoFlashMessageKeys`, `NoLogKeys`) are `never`. I dont know why. I could not fix it.

Reply:
Maybe it was some TS server error, but on my side it's working. 

![image](https://github.com/shopsys/shopsys/assets/11465991/5ea2825c-f3a0-430f-aacc-ef79924689ea)

Reply:
Also please remove `export` from this `ApplicationErrors` constant. It is used only internally in this file.

Reply:
I was curious so I tried but no success neither. So let's leave it since those typings for `getErrorMessageTranslationString` is great.